packages/fetchai/skills/aries_faber/handlers.py

```diff
@@ -0,0 +1,214 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This package contains the handlers for the faber_alice skill."""
+
+import json
+from typing import Dict, Optional, cast
+
+from aea.configurations.base import ProtocolId, PublicId
+from aea.mail.base import Envelope, EnvelopeContext
+from aea.protocols.base import Message
+from aea.protocols.default.message import DefaultMessage
+from aea.protocols.default.serialization import DefaultSerializer
+from aea.skills.base import Handler
+
+from packages.fetchai.protocols.http.message import HttpMessage
+from packages.fetchai.protocols.http.serialization import HttpSerializer
+
+HTTP_PROTOCOL_PUBLIC_ID = PublicId("fetchai", "http", "0.1.0")
+DEFAULT_PROTOCOL_PUBLIC_ID = PublicId("fetchai", "default", "0.1.0")
+
+OEF_CONNECTION_PUBLIC_ID = PublicId("fetchai", "oef", "0.1.0")
+
+DEFAULT_ADMIN_HOST = "127.0.0.1"
+DEFAULT_ADMIN_PORT = 8021
+SUPPORT_REVOCATION = False
+
+
+class AriesDemoHTTPHandler(Handler):
+    """This class represents faber's handler for default messages."""
+
+    SUPPORTED_PROTOCOL = HttpMessage.protocol_id  # type: Optional[ProtocolId]
+
+    def __init__(self, **kwargs):
+        """Initialize the handler."""
+        self.admin_host = kwargs.pop("admin_host", DEFAULT_ADMIN_HOST)
+        self.admin_port = kwargs.pop("admin_port", DEFAULT_ADMIN_PORT)
+        self.alice_id = kwargs.pop("alice_id")
```

Comment:
should have a default value as well

Reply:
What do you suggest as a default value for it?

Reply:
up to you tbh,

Reply:
Done